### ISSUE TITLE
Are these going to be packaged into functions?

### ISSUE THREAD

**Issue by muschellij2** [ntustison/ANTsRNet#2]
This seems like a great resource for ANTsR, but will be pretty hard to use for most users.  Is there a plan to package the models and the processing (image to patches/slices and reorientation) into functions and then distribute them or just give these as examples as such?

I think adding the functionality to pass in an image or image filename and get back and image/filename of the output would be a great addition.

**Comment by ntustison** [comment]
Admittedly, this isn't an optimal packaging of the examples.  I'm still thinking about it.  But, just to be clear, are you aware of the main package [ANTsRNet](https://github.com/ANTsX/ANTsRNet/tree/master/R)?

**Comment by stnava** [comment]
yes - i think that parts will move to antsrnet as they mature - this is intended for somewhat raw development

**Comment by muschellij2** [comment]
Yes - I have seen that and that's how I got to these.  I'm just wondering how we could package the trained `h5` models into a package effectively (and lightweight).

**Comment by stnava** [comment]
we are looking at git-annex (or via datalad) as a possibility.

but this is still in the nascent stage.


brian
